apiary.apib

```diff
@@ -36,44 +37,79 @@ current session status.
 
 + Response 201 (application/vnd.api+json)
 
-        {
-            "links": {
-                "self": "https://example.com/session"
-            },
-            "data": {
-                "id": "sessionid123",
-                "type": "session"
-            },
-            "related": {
-                "account": {
-                    "links": {
-                        "related": "https://example.com/session/account"
-                    },
-                    "data": {
-                        "id": "abc1234",
-                        "type": "account"
+    + Headers
+
+            Location: https://example.com/session
+
+    + Body
+
+            {
+                "links": {
+                    "self": "https://example.com/session"
+                },
+                "data": {
+                    "id": "sessionid123",
+                    "type": "session",
+                    "relationships": {
+                        "account": {
+                            "links": {
+                                "related": "https://example.com/session/account"
+                            },
+                            "data": {
+                                "id": "abc1234",
+                                "type": "account"
+                            }
+                        }
                     }
                 },
-                "profile": {
-                    "links": {
-                        "related": "https://example.com/session/account/profile"
+                "included": [{
+                    "id": "abc1234",
+                    "type": "account",
+                    "attributes": {
+                        "username": "jane-doe"
                     },
-                    "data": {
-                        "id": "profile",
-                        "type": "accountProfile"
+                    "relationships": {
+                        "profile": {
+                            "links": {
+                                "related": "https://example.com/session/account/profile"
+                            },
+                            "data": {
+                                "id": "abc1234-profile",
+                                "type": "accountProfile"
+                            }
+                        }
                     }
-                }
+                }]
             }
```

Comment:
The way the `account` and `profile` links (and ids) are added to the response changed quite a lot, see why here  http://discuss.jsonapi.org/t/is-it-ok-to-add-multiple-levels-of-nested-relationships-in-the-relationships-field/167. 

@gr2m, I think we discussed about responses being too verbose, most of the fields here can be left out, for example, we could also just give out the `session.data` part, depends on how much information we think is useful for the client.